### PR TITLE
fix(components): [el-upload] jitter when remove file in the list

### PR DESCRIPTION
**Pull request by msidolphin**
before

https://user-images.githubusercontent.com/26672484/157407053-709d9fa6-7b1b-420a-bdc7-b90fc9c9adca.mp4

after

https://user-images.githubusercontent.com/26672484/157408000-d8cb1307-5f1f-4934-a84c-5e71ea6bce63.mp4

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by JeremyWuuuuu** [comment]
#6508 is refactoring this component, can we wait for it to see if we will still have this issue?

**Comment by sxzz** [comment]
@JeremyWuuuuu There still seems to be this problem.

**Comment by sxzz** [comment]
@msidolphin Could you please resolve conflicts?

**Review by YunYouJun — CHANGES_REQUESTED** [review]

File: packages/theme-chalk/src/upload.scss [hunk 1]
```diff
@@ -141,23 +141,19 @@
...
 
-    &:first-child {
-      margin-top: 10px;
-    }
-
     & .#{$namespace}-icon--upload-success {
       color: var(--el-color-success);
     }
 
     .#{$namespace}-icon--close {
```
[the remark this answers is not in the document]
Comment (msidolphin) [reply packages/theme-chalk/src/upload.scss]: > It seems to cause error position of photo wall close button.

image

fix～

**Comment by msidolphin** [comment]
> @msidolphin Could you please resolve conflicts?

done

**Review by jw-foss — APPROVED** [review]

**Review by jw-foss — APPROVED** [review]

**Review by YunYouJun — APPROVED** [review]

**Pull request merged**